tests/test_auto3dseg_ensemble.py

```diff
@@ -34,27 +34,25 @@
 
 _, has_tb = optional_import("torch.utils.tensorboard", name="SummaryWriter")
 
+num_images_perfold = max(torch.cuda.device_count(), 4)
+num_images_per_batch = 2
+
 fake_datalist: dict[str, list[dict]] = {
     "testing": [{"image": "val_001.fake.nii.gz"}, {"image": "val_002.fake.nii.gz"}],
     "training": [
-        {"fold": 0, "image": "tr_image_001.fake.nii.gz", "label": "tr_label_001.fake.nii.gz"},
```

Comment:
Hi @wyli , What I saw such cases with MONAI 1.0.1 was that dataset partitioning will fail first:
https://github.com/Project-MONAI/research-contributions/blob/b37ed82d6a8504adc2a27a2cf00aa0656626d60b/auto3dseg/algorithm_templates/dints/scripts/train.py#L111
Then the torchrun process is disrupted because it is unable to complete the work.